schemas/schema.appointment.js

```diff
@@ -14,13 +14,10 @@ const definition = {
 		"type": Date,
 		"required": true,
 	},
-	"time": { // Format -> HH:mm
-		"type": String,
-		"required": true,
-	},
 	"treatmentMaster": String,
 	"status": {
-		"type": Boolean,
+		"type": String,
+		"enum": ["No show", "confirmed", "arrived", "cancelled"],
```

Comment:
Make sure that statuses have upper case first letters.
`confirmed` -> `Confirmed` etc.